articles/hosted-pages/login/auth0js/v8/index.md

```diff
@@ -0,0 +1,174 @@
+---
+description: How to Use the Auth0.js v8 with the Hosted Login Page 
+---
+# Using Auth0.js v8 in the Hosted Login Page
+
+Within the Hosted Login page, you can use the the [Auth0.js v8 SDK](/libraries/auth0js/v8), instead of [Lock](/libraries/lock), to perform authentication using a custom UI (you can also use Auth0.js _in addition_ to Lock, for authentication or user management tasks).
+
+Auth0.js is the current version of the SDK, and is the recommended version for projects that are able to use it. You can read the Reasons to Migrate and the API Auth sections of the [Auth0.js Migration Guide](/libraries/auth0js/v8/migration-guide#reasons-to-migrate) to learn whether version 8 of Auth0.js is appropriate for your applications if you are uncertain.
+
+## Auth0.js Template for Hosted Login Page
+
+You can start out with a basic template that will provide you with a working, ready-to-use example of a custom UI using Auth0.js v8 in the Hosted Login Page. In the [dashboard](${manage_url}), go to Hosted Pages, and then to the Login page section. At the top of the code editor for the page contents, you'll see a dropdown, titled "Default Templates". Here you can choose from Lock, Lock Passwordless, or Custom UI. Choosing "Custom UI" will give you an example of an entire page using Auth0.js, including the custom UI styling. You can use this template as an example to get you started.
```

Comment:
Also, the last two sentences are just repeating what you have already said. Just tell them to choose `Custom UI` for this scenario.